package.json

```diff
@@ -5,14 +5,15 @@
     "start": "node ./bin/www",
     "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha ./test/**/*.js --exit",
```

Comment:
Running the command locally, there are two failing tests. Not sure whether they're already failing on develop. It appears these they're failing differently than the ones that fail on develop.

Reply:
Yeah they would have been failing before as well - it looks because of the issue described in my PR summary.